optuna/samplers/_brute_force.py

```diff
@@ -3,12 +3,8 @@
 from dataclasses import dataclass
 import decimal
 from typing import Any
-from typing import Dict
 from typing import Iterable
-from typing import List
-from typing import Optional
 from typing import Sequence
```

Comment:
`Iterable` and `Sequence` should be imported from `collections.abc`.
https://peps.python.org/pep-0585/
```suggestion
from collections.abc import Iterable
from collections.abc import Sequence
```

Reply:
Thank you for your comment! I revised the code.